### PR TITLE
T/check username

### PR DESCRIPTION
**Pull request by bluemaex**
add a second factor to check when requesting token

### CODE REVIEW

File: src/easybib-api-client.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ module.exports = function($q, $http, store, $timeout, easyBibApiAccessUrl) {
   self.$q = $q;
   self.$http = $http;
   self.accessTokenHttpOptions = {};
+  if ($injector.has('easyBibApiAccessUsername')) {
+    self.checkUsername = $injector.get('easyBibApiAccessUsername');
```
Comment (sashless) [inline src/easybib-api-client.js]: The naming is misleading. By the naming and the usage i have doubts about what this function tries to do. Seems like it returns true/false like a `check` function would do but its just a getter. I would make that clear through naming like `getUsername()` or just `username()`. Despite that the functionality looks like it works

Comment (bluemaex) [inline src/easybib-api-client.js]: Good point. Will rename ``checkUsername`` into ``getUsername`` to make it more obvious.

Comment (sashless) [inline src/easybib-api-client.js]: Nice :+1:

**Comment by sashless** [comment]
:+1:

**Pull request merged**